framework/views/errorHandler/exception.php

```diff
@@ -12,16 +14,16 @@
 
     <title><?php
         $name = $handler->getExceptionName($exception);
-        if ($exception instanceof \yii\web\HttpException) {
```

Comment:
This is what `PHPCBF` thinks it should look like. Haven't reviewed this myself yet.